addons/binding/org.openhab.binding.exec/README.md

```diff
@@ -16,31 +16,42 @@ Note that the commands are executed in the context and with the privileges of th
 
 The `command` Thing requires the command to execute on the shell. Optionally one can specify:
 
-- a transformation to apply on the execution result, 
-- an interval, in seconds, the command will be repeatedly executed, 
-- a time-out, in seconds, the execution of the command will time out, and lastly, 
-- a boolean parameter to make the command execute immediately every time the state of the input channel has changed. 
+- [transform] a transformation to apply on the execution result, 
+- [interval] an interval, in seconds, the command will be repeatedly executed, 
+- [timeput] a time-out, in seconds, the execution of the command will time out, and lastly, 
+- [runOnInput] a boolean parameter to make the command execute immediately every time the state of an input channel is set by a Command 
```

Comment:
I either don't understand your response and/or you didn't understand my comments. Maybe you are using words "last" and "previous" in some special way???

It would help me if you don't refer to channels without specifying *which* channel you mean. I don't know of anything in my suggestions that would require implementing handleUpdate.